pawan/html/practice_assignment/Programming_Exercises4.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Resume</title>
+        <style> 
+            table,th,td{
+            border: 2px solid black;
+            }
+        </style>
+    </head>
+    <body>
```

Comment:
If you would have understand the importance of "div", you could have use that in this code.

Reply:
i will do that